### PR TITLE
TRUNK-295: Removed addProvyPrivilege and removeProxyPrivilege methods

### PR DESCRIPTION
**Pull request by SsebaddukaG**
https://issues.openmrs.org/browse/TRUNK-295

### CODE REVIEW

**Comment by HerbertYiga** [comment]
some failing travis here!

**Comment by SsebaddukaG** [comment]
@HerbertYiga yeah, still working on asap.

File: api/src/main/java/org/openmrs/api/context/Context.java [hunk 1]
```diff
@@ -1061,12 +1048,10 @@ public static void checkCoreDataset() {
 			log.error("Error while setting core roles for openmrs system", e);
 		}
 		finally {
-			Context.removeProxyPrivilege(PrivilegeConstants.MANAGE_ROLES);
```
Comment (dilanthas) [inline api/src/main/java/org/openmrs/api/context/Context.java]: No point of having an empty finally block. Please remove that too..

File: api/src/main/java/org/openmrs/api/context/Context.java [hunk 2]
```diff
@@ -1086,13 +1071,10 @@ public static void checkCoreDataset() {
 			log.error("Error while setting core privileges", e);
 		}
 		finally {
-			Context.removeProxyPrivilege(PrivilegeConstants.MANAGE_PRIVILEGES);
```
Comment (dilanthas) [inline api/src/main/java/org/openmrs/api/context/Context.java]: Another empty finally block

File: api/src/main/java/org/openmrs/api/handler/PatientDataVoidHandler.java [hunk 3]
```diff
@@ -60,12 +60,11 @@ public void handle(Patient patient, User voidingUser, Date voidedDate, String vo
 			}
 		}
 
-		Context.addProxyPrivilege(PrivilegeConstants.EDIT_COHORTS);
+		
 		try {
 			Context.getCohortService().notifyPatientVoided(patient);
 		}
 		finally {
-			Context.removeProxyPrivilege(PrivilegeConstants.EDIT_COHORTS);
```
Comment (dilanthas) [inline api/src/main/java/org/openmrs/api/handler/PatientDataVoidHandler.java]: Another one

**Comment by dilanthas** [comment]
There are some empty finally blocks. Please remove them too

File: api/src/main/java/org/openmrs/api/context/Context.java (hunk 1 above)
Comment (SsebaddukaG) [reply api/src/main/java/org/openmrs/api/context/Context.java]: Am working on it.

File: api/src/main/java/org/openmrs/api/handler/PatientDataVoidHandler.java (hunk 3 above)
Comment (SsebaddukaG) [reply api/src/main/java/org/openmrs/api/handler/PatientDataVoidHandler.java]: Still working on the travis failure and then clean up the code.

**Comment by sherrif10** [comment]
are u still blocked by travis @SsebaddukaG

**Pull request closed without merging**